widget/slider.go

```diff
@@ -523,17 +523,18 @@ func (s *sliderRenderer) Layout(size fyne.Size) {
 
 // MinSize calculates the minimum size of a widget.
 func (s *sliderRenderer) MinSize() fyne.Size {
-	dia := s.slider.buttonDiameter(s.slider.Theme().Size(theme.SizeNameInlineIcon))
-	s1, s2 := minLongSide+dia, dia
+	th := s.slider.Theme()
+	pad := th.Size(theme.SizeNameInnerPadding)
+	dia := s.slider.buttonDiameter(th.Size(theme.SizeNameInlineIcon))
```

Comment:
Switching the order of this and the line below it allows you to only invoke `th.Size(theme.SizeNameInlineIcon)` once and just pass the computed value to buttonDiameter

Reply:
Thanks for this, updated accordingly